packages/suite/src/views/wallet/receive/components/FreshAddress.tsx

```diff
@@ -141,15 +140,13 @@ export const FreshAddress = ({
 
     if (!account || !firstFreshAddress) return null;
 
-    const getAddressValue = (address?: AccountAddress) => {
-        if (!address) {
-            return <Translation id="RECEIVE_ADDRESS_LIMIT_EXCEEDED" />;
-        }
-
-        return `${address.address.substring(0, 20)}`;
-    };
+    const addressValue = `${firstFreshAddress.address.substring(0, 20)}`;
 
-    const addressValue = getAddressValue(firstFreshAddress);
+    // On CJ account, disallow to reveal more than the first receive address until it is used
```

Comment:
nitpick: I always find it better to explain why rather than what in comment. 

something like?
```suggestion
    // On coinjoin account, disallow to reveal more than the first receive address until it is used, because discovery of coinjoin account relies on assumption that user uses his first address first.
```

Reply:
Why do you hate CJ? :sob: 
https://github.com/trezor/trezor-suite/pull/8301/commits/c79285caa933f709bb767fb5356e32b1a5422e27

Reply:
😆 I like CJ but I hate shortcuts, because I'm stupid and often can't figure out what they mean...